docs/pages/web/slimbanner.js

```diff
@@ -445,23 +444,46 @@ Combine SlimBanners with other components like [Callouts](/web/callout) or [Upse
           />
         </MainSection.Subsection>
         <MainSection.Subsection
-          description="The SlimBanner message can be complemented with a helper Link."
-          title="helperLink"
+          description={`The \`message\` prop accepts both strings and [Text](/Text). Use strings for simple messages without any visual style. SlimBanner will handle the message style and the adherence to design guidelines. If a message with more complex style is required, such as bold text or inline links, use Text to inline text and links within the same Text component.
+
+The SlimBanner \`message\` string can be complemented with a \`helperLink\`. When passing a Text component, \`helperLink\` isn't rendered to prevent unnecessary visual load.
+
+Due to localization constrains, the contents of \`message\` and \`helperLink\` cannot belong to the same sentence. They must be independent sentences separated by a period.
```

Comment:
typo: "Due to localization _constraints_…"

Maybe add one more sentence at the end? We want to be a bit forceful here. Perhaps something like "Do not attempt to construct a compound sentence using `message` and `helperLink`."

Reply:
DONE